apps/organisations/templates/a4_candy_organisations/communication_form_social_media.html

```diff
@@ -24,8 +24,21 @@ <h1>{% translate "Create Content" %}</h1>
             {% include 'a4_candy_contrib/includes/form_field.html' with field=content_form.title %}
             {% include 'a4_candy_contrib/includes/form_field.html' with field=content_form.description %}
             {% include 'a4_candy_contrib/includes/form_field.html' with field=content_form.image %}
+            <div class="checkbox-btn">
+                <label class="checkbox-btn__label--light" for="logo_1">
+                    <input class="checkbox-btn__input" type="checkbox" id="logo_1" value="add a+ logo" checked>
+                    <span class="checkbox-btn__text--icon">{% translate 'Add a+ logo' %}</span>
+                </label>
+            </div>
+            <div class="checkbox-btn">
+                <label class="checkbox-btn__label--light" for="logo_2">
+                    <input class="checkbox-btn__input" type="checkbox" id="logo_2" value="add own logo" checked>
+                    <span class="checkbox-btn__text checkbox-btn__text--icon">{% translate 'Add logo' %}</span>
+                </label>
+            </div>
```

Comment:
@Rineee should I add these as a form_field include? I don't really know?!

Reply:
No, I think it is fine like this, cool! 
Should we maybe grey out the 'add logo' if there is no organisation logo?

Reply:
yes, good plan, you said that before but I forgot